config/routes.rb

```diff
@@ -65,6 +65,11 @@ def matches?(request)
       end
     end
 
+    resources :unsubscribe,
+      path: "/unsubscribe/:resource_class",
+      constraints: {resource_class: /reminders/},
+      only: [:create, :show]
+
```

Comment:
`resource_class` is not currently used. buy has been added so can the extended to support unsubscribing of other resource in the future easily if needed